test/lint/lint-circular-dependencies.sh

```diff
@@ -65,7 +59,6 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "llmq/signing -> net_processing -> llmq/signing"
     "llmq/signing_shares -> net_processing -> llmq/signing_shares"
     "logging -> util/system -> logging"
-    "logging -> util/system -> random -> logging"
```

Comment:
additional changes done as reported by linter that the dependency is no longer present